lib/ansible/modules/files/unarchive.py

```diff
@@ -752,7 +752,7 @@ def can_handle_archive(self):
             return False, 'Command "%s" could not handle archive.' % self.cmd_path
         # Errors and no files in archive assume that we weren't able to
         # properly unarchive it
-        return False, 'Command "%s" found no files in archive.' % self.cmd_path
+        return False, 'Command "%s" found no files in archive. Unarchiving empty archive file is not supported yet.' % self.cmd_path
```

Comment:
I don't think we should use the word `yet`, as that indicates we have already made the decision that it should, and we are actively working on changing it.

Also, we need to work a little on the grammar.

"Empty archive files are not supported"